AMaDda/Screens/Adding/AddingViewController.swift

```diff
@@ -0,0 +1,135 @@
+        return label
+    }()
+    private let profileImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.btnProfile
+        imageView.tintColor = .lightGray
+        return imageView
+    }()
+    private let plusIcon: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = ImageLiterals.icPlus
+        imageView.tintColor = .gray
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+    private let createNicknameLabel: UILabel = {
+        let label = UILabel()
+        label.text = "어떻게 부르고 싶나요?"
+        label.font = UIFont.systemFont(ofSize: 20, weight: .regular)
+        return label
+    }()
+    private let nickNameTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "예시) 아버지, 엄마, 세젤예 우리엄마"
+        return textField
+    }()
+    private let underLineView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .lightGray
+        return view
+    }()
+    private let textFieldLimitLabel: UILabel = {
+        let label = UILabel()
+        label.text = "0/10"
+        label.textColor = .lightGray
+        return label
+    }()
+    private let addButton: CommonButton = {
+        let button = CommonButton()
+        button.title = "추가하기"
+        return button
+    }()
+    // MARK: - life cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        configureAddSubView()
+        configureConstraints()
+    }
+
+    // MARK: - configure
+    private func configureUI() {
+        view.backgroundColor = .systemBackground
+    }
+    
+    private func configureAddSubView() {
+        view.addSubview(addingTitleLabel)
+        view.addSubview(profileImageView)
+        view.addSubview(plusIcon)
+        view.addSubview(createNicknameLabel)
+        view.addSubview(nickNameTextField)
+        view.addSubview(underLineView)
+        view.addSubview(textFieldLimitLabel)
+        view.addSubview(addButton)
+    }
+    
+    private func configureConstraints() {
+        addingTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            addingTitleLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            addingTitleLabel.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor,constant: Size.leadingTrailingPadding),
+        ])
+        profileImageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            profileImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            profileImageView.topAnchor.constraint(equalTo: addingTitleLabel.bottomAnchor, constant: 37),
+            profileImageView.widthAnchor.constraint(equalToConstant: 100),
+            profileImageView.heightAnchor.constraint(equalToConstant: 100),
+        ])
```

Comment:
제가 코드를 짜본 결과, `NSLayoutConstraint.active([  ~~~ ])` 안에 모든 코드들을 전부 넣을 수 있던데, 
이렇게 나눈 이유는 일부러 가독성을 위해서 나눈건가요?

예를 들어 아래와 같이 바꿔도 코드는 가독성이 떨어지는 걸까요? (사실 제가 봐도 가독성은 떨어져보이긴 합니다만.. 혹시 몰라 여쭤봅니다~)
```suggestion
private func configureConstraints() {
addingTitleLabel.translatesAutoresizingMaskIntoConstraints = false
profileImageView.translatesAutoresizingMaskIntoConstraints = false

NSLayoutConstraint.activate([
            addingTitleLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
            addingTitleLabel.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor,constant: Size.leadingTrailingPadding),
            
            profileImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
            profileImageView.topAnchor.constraint(equalTo: addingTitleLabel.bottomAnchor, constant: 37),
            profileImageView.widthAnchor.constraint(equalToConstant: 100),
            profileImageView.heightAnchor.constraint(equalToConstant: 100)
])
```

Reply:
제 생각에는 나눠서 하는게 NSLayoutConstraints에 포함되는 코드들이
하나의 component와 관련된 constraint라는 점에서 가독성이 더 늘어나는 것 같습니다!

```swift
// addingTitleLabel 관련 constraints는 이 아래부터 시작
addingTitleLabel.translatesAutoresizingMaskIntoConstraints = false
NSLayoutConstraint.activate([
    addingTitleLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
    addingTitleLabel.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor,constant: Size.leadingTrailingPadding),
])

// profileImageView 관련 constraints는 이 아래부터 시작
profileImageView.translatesAutoresizingMaskIntoConstraints = false
NSLayoutConstraint.activate([
    profileImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
    profileImageView.topAnchor.constraint(equalTo: addingTitleLabel.bottomAnchor, constant: 37),
    profileImageView.widthAnchor.constraint(equalToConstant: 100),
    profileImageView.heightAnchor.constraint(equalToConstant: 100),
])

```

요렇게 뭔가 구간을 나눠주는 것 같기도 했습니다!